src/app/problem/edit/description/MarkdownEditor.js

```diff
@@ -0,0 +1,146 @@
+import '../../ProblemEditor.css';
+import React, {useState} from "react";
```

Comment:
Where did you use `ProblemEditor.css` ?
Don't use `ProblemEditor.css` when `MarkdownEditor` is a common component.